test.js

```diff
@@ -159,26 +189,39 @@ ${Object.values(commands).map(({ name, description }) => `    ${name.padEnd(21,
 const runTest = async () => {
   const parameters = process.argv.slice(2);
   const hasParameters = Boolean(parameters.length);
-  const [ commandName ] = parameters;
+  let [ passedArgs = '' ] = parameters;
+  const [ commandName ] = passedArgs.split(' ');
   const command = commands[commandName];
   const isCommandNotFound = !command;
 
+  // Read the input for passed arguments that arent command names
+  passedArgs = passedArgs.trim().replaceAll('--', '').toLowerCase().split(' ').filter(name => isCommandNotFound || name !== commandName);
+
+  // Update argumentValues array for later use while checking if the command is valid
+  const paramsRecognised = passedArgs.every(passedArg => {
+    const passedArgParts = passedArg.trim().split('=');
+    argumentValues[passedArgParts[0]] = passedArgParts[1];
```

Comment:
does not override default `testfiles` if not supplied, default `testfiles` is therefore always supplied to both cypress and jest, but is not specific enough for either.